test/Origin.events.js

```diff
@@ -131,6 +131,22 @@ test.describe("Origin events", () => {
       });
     });
 
+    test.it("should emit a cleanAny event containing data about the cleaned source", () => {
+      let eventData;
+      testOrigin.onCleanAny(data => {
```

Comment:
Stubbing the callback function could be a better and standard approach.

Reply:
Done